app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -102,19 +107,32 @@ public ViewModel(final @NonNull Environment environment) {
 
       final Observable<Void> refreshSurvey = Observable.merge(this.refresh, this.resume).share();
 
-      this.currentUser.loggedInUser()
+      final Observable<User> loggedInUser = this.currentUser.loggedInUser();
+
+      loggedInUser
         .compose(takeWhen(refreshSurvey))
-        .switchMap(__ -> this.client.fetchUnansweredSurveys().compose(neverError()).share())
+        .switchMap(__ -> this.apiClient.fetchUnansweredSurveys().compose(neverError()).share())
         .compose(bindToLifecycle())
         .subscribe(this.surveys);
 
+      loggedInUser
+        .compose(takeWhen(refreshSurvey))
```

Comment:
Just curious why this is called `refreshSurvey?`

Reply:
not sure,  I think a better name could be `resumeOrRefresh`